src/components/doc-footer/doc-footer.tsx

```diff
@@ -10,10 +11,11 @@ const DocFooter: FC<DocFooterProps> = ({ feedbackFormLink }) => (
   <div className={styles.docFooter}>
     <SectionName>Feedback</SectionName>
     <div className={styles.text}>
-      Help us improve this pattern by providing feedback, asking questions, and leaving any other comments on
-      <a href={feedbackFormLink} className={styles.link}>
+      Help us improve this pattern by providing feedback, asking questions, and leaving any other comments on{' '}
+      <Link href={feedbackFormLink} withoutSpacing>
```

Comment:
Does it open the link in the same tab or a new tab?

Reply:
Another tab by default

Reply:
Not sure about this case specifically but in other areas where we replace `<a>` with a link it now opens in a new tab when you navigate inside storybook, for example when clicking on the table of content